client/state/memberships/coupon-list/test/reducer.js

```diff
@@ -0,0 +1,55 @@
+import deepFreeze from 'deep-freeze';
```

Comment:
Thanks for adding state tests for the coupon list subtree. FWIW this could have been its own PR since this one is already quite large.